src/AccessibilityInsights/MainWindowHelpers/EventsMode.cs

```diff
@@ -89,5 +91,23 @@ private void StartEventsWithLoadedData(string path)
 
             PageTracker.TrackPage(this.CurrentPage, this.CurrentView.ToString());
         }
+
+        /// <summary>
+        /// Deserialize EventMessages from JSON file. 
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        public static List<EventMessage> LoadEventMessages(string path)
```

Comment:
>LoadEventMessages [](start = 41, length = 17)

You could probably replace this with a call to SetupLibrary.FileHelpers.LoadDataFromJSON<List<EventMessage>>(path). That's the method that ConfigurationModel ultimately uses for its serialization (it's in the SetupLibrary because it's also used by the VersionSwitcher)